backend/src/main/java/se/chalmers/dat265/group1/model/dto/PhysicalPrintSlm.java

```diff
@@ -0,0 +1,8 @@
+package se.chalmers.dat265.group1.model.dto;
+
+import se.chalmers.dat265.group1.model.PhysicalPrint;
+
+public class PhysicalPrintSlm extends PhysicalPrint {
+    private String path = "";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "path" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)